debugger/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -63,7 +62,7 @@ trait Parse {
 object Parse {
   implicit val logger: Logger[IO] = Slf4jLogger.getLogger
 
-  case class Exception(diagnostics: List[Diagnostic])
+  case class Exception(diagnostics: List[org.apache.daffodil.sapi.Diagnostic])
```

Comment:
Why not import sapi.Diagnostic?

Reply:
`lib.api.Diagnostic` is already in scope, but I guess I could create an alias via `import`.

FYI `lib.api.Diagnostic` is returned via `runtime1.processors.parsers.PState`, which is the data structure available via the `Debugger` interface, but `sapi.Diagnostic` is available via the `sapi.ParseResult` provided by `sapi.DataProcessor.parse()`.